ProjectBank/Client/Shared/FormComponent.razor

```diff
@@ -0,0 +1,57 @@
+<EditForm Model="@Project" OnValidSubmt="@HandleValidSubmit">
+    <DataAnnotationsValidator />
+    <ValidationSummary />
+
+    <div class="mb-3">
+        <label class="form-label">Title</label>
+        <InputText id="title" class="form-control" @bind-Value="Project.Title" />
+    </div>
+
+    <div class="mb-3">
+        <label class="form-label">Description</label>
+            <InputTextArea id="description" class="form-control" @bind-Value="Project.Description" />
+    </div>
+
+    <div class="mb-3">
+        <label class="form-label">Tags</label>
+            <InputText id="tags" class="form-control" @bind-Value="@TagListString"/>
+    </div>
+
+    <button type="submit" class="btn btn-success" onclick="@HandleValidSubmit">@ButtonName</button>
+</EditForm> 
+
+@code {
+    [Parameter]
+    public Project Project {get; set;}
+
+    [Parameter]
+    public string ButtonName {get; set;}
+
+    private string TagListString = "";
+    
+    //receiving a list of tags that have to be displayed
```

Comment:
I'm not sure turning a list og Tags into string should be something our frontend takes care of? 
We have similar functionality in our ProjectRepository. 
If it should be the frontend - Project should probably be one of our DTOs.

Reply:
It does sound reasonable, I don't think i have the overview of how it will work together to determine what should be done, so any suggestions on how this should be resolved?